apps/hash-ai-worker-ts/src/activities/flow-activities/shared/infer-facts-from-text/infer-entity-facts-from-text.ts

```diff
@@ -278,9 +283,9 @@ export const inferEntityFactsFromText = async (params: {
       ],
     });
   } else if (llmResponse.status !== "ok") {
-    throw new Error(
-      `Failed to get response from LLM: ${stringify(llmResponse)}`,
-    );
+    return {
+      facts: params.retryContext?.previousValidFacts ?? [],
+    };
```

Comment:
You'll see when resolving conflicts, but I've addressed this by retrying the request in the agent here rather than returning the existing facts. This is to account for scenarios where the first response from the LLM leads to a schema validation error that it cannot recover from (which is possible with `haiku`), in a way that doesn't result in 0 facts being observed as easily.